src/components/VerticalCard/VerticalCard.tsx

```diff
@@ -49,50 +53,61 @@ export const VerticalCard = ({
       className={cnb(styles.root, className)}
       {...props}
     >
-      {imageSrc && (
-        <div className={styles.imageWrapper}>
-          <img
-            alt=""
-            src={getProcessedImage(imageSrc, '600x600', imageFocus)}
-            className={styles.image}
-          />
-        </div>
-      )}
-      {heading && (
-        <Heading
-          as={headingLevel}
-          font={headingFont}
-          size={isSmallHeading ? 3 : 4}
-          leading="tight"
-          className={styles.heading}
-        >
-          {(!ctaLabel && (link || href))
-            ? (
-              <CtaLink sbLink={link} href={href} className={styles.headingLink}>
-                {heading}
-              </CtaLink>
-            ) : heading}
-        </Heading>
-      )}
-      {tabColor && (
+      <div className={styles.cardWrapper}>
+        {imageSrc && (
+          <div className={styles.imageWrapper}>
+            <img
+              alt=""
+              src={getProcessedImage(imageSrc, '600x600', imageFocus)}
+              className={styles.image}
+            />
+          </div>
+        )}
+        {heading && (
+          <Heading
+            as={headingLevel}
+            size={isSmallHeading ? 3 : 4}
+            leading="tight"
+            className={cnb(styles.heading(!!tabColor), accentBorderColors[tabColor])}
+          >
+            {(!ctaLabel && (link || href))
+              ? (
+                <CtaLink sbLink={link} href={href} className={styles.headingLink}>
+                  {heading}
+                </CtaLink>
+              ) : heading}
+          </Heading>
+        )}
+        {/* {tabColor && (
         <div className={cnb(styles.tab, accentBgColors[tabColor])} />
-      )}
-      {body && (
-        <Paragraph variant="card" noMargin>{body}</Paragraph>
-      )}
-      {ctaLabel && (link || href) && (
-        <CtaLink
-          variant="solid"
-          curve="br"
-          icon="arrow-right"
-          animate="right"
-          srText={ctaSrText}
-          sbLink={link}
-          href={href}
-          className={styles.ctaLink}
-        >
-          {ctaLabel}
-        </CtaLink>
+      )} */}
+        {body && (
+          <Paragraph variant="card" noMargin>{body}</Paragraph>
+        )}
+        {ctaLabel && (link || href) && (
+          <CtaLink
+            variant="solid"
+            curve="br"
+            icon="arrow-right"
+            animate="right"
+            srText={ctaSrText}
+            sbLink={link}
+            href={href}
+            className={styles.ctaLink}
+          >
+            {ctaLabel}
+          </CtaLink>
+        )}
+      </div>
+      {/* Display max 3 topic tags; TODO: Sort tags after clients decide on order */}
+      {taxonomy?.length > 0 && (
+        <ul className={styles.taxonomy(!!tabColor)}>
+          {taxonomy.slice(0, 3).map((item) => (
```

Comment:
Just came out of meeting - client would like to see 3 max topic tags under Story cards, but sort order is TBD